wandb/sdk/wandb_run.py

```diff
@@ -1888,6 +1889,21 @@ def _show_history(self) -> None:
                 history_lines += format_str.format(*row)
             wandb.termlog(history_lines)
 
+    def _show_local_warning(self) -> None:
+        if not self._poll_exit_response or not self._poll_exit_response.local_info:
+            return
+
+        if self._settings._offline:
+            return
+
+        if self._settings.is_local:
+            local_info = self._poll_exit_response.local_info
+            latest_version, out_of_date = local_info.version, local_info.out_of_date
+            if out_of_date:
+                wandb.termwarn(
+                    f"Upgrade to W&B Local {latest_version} to get the latest features. Learn more: http://wandb.me/local-upgrade"
```

Comment:
In some cases, the server doesn't know what the latest version is, right? In those cases, would this message look busted?

Reply:
I think I guard against it with default value for `latest_version` see:
https://github.com/wandb/client/pull/2513/files#diff-d2120fc2c9f40441f52d074ef89f635a83c01ebba43bb4092130b1058fdc3d9cR1031-R1048]

Reply:
So then it would print "Upgrade to W&B Local latest to get the latest features..." ?

It's a nitpick, but it would be nice to say "Upgrade to the latest version of W&B Local to get the latest...." in the case where we don't know the actual version number.